### PR TITLE
schachte docker aws

### PR DESCRIPTION
**Pull request by Schachte**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "start:test": "ace start --env=test",
     "test:api": "mocha src/main/webapp/intrigue-api/test",
     "pretest": "ace format && eslint src",
-    "test": "ace bundle --env=test && ace test",
```
Comment (Schachte) [inline package.json]: Revert

File: .travis.yml [hunk 2]
```diff
@@ -8,19 +11,23 @@ before_install:
   - export PATH="$HOME/.yarn/bin:$PATH"
 script:
   - set -e
-  - mvn clean install
   - yarn install --frozen-lockfile
-  - yarn test
```
Comment (Schachte) [inline .travis.yml]: revert

File: .travis.yml [hunk 3]
```diff
@@ -8,19 +11,23 @@ before_install:
...
-  - yarn build
+  - yarn build-bundles
```
Comment (djblue) [inline .travis.yml]: I think you can update the command to build bundles instead.

File: .travis.yml [hunk 4]
```diff
@@ -8,19 +11,23 @@ before_install:
...
   - set +e
 branches:
   only:
   - master
+  - docker-aws
```
Comment (djblue) [inline .travis.yml]: Are we keeping this?

File: dockerDeploy [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+docker login -u $DOCKER_HUB_USER -p $DOCKER_HUB_PASSWORD
+
+currentHash=$(git rev-parse --short=8 HEAD)
+imageName="benandryan1/revelio2:$currentHash"
```
Comment (djblue) [inline dockerDeploy]: Are we keeping this name? :laughing:

**Pull request merged**